source/nodejs/adaptivecards-designer/src/designer-peers.ts

```diff
@@ -1664,19 +1670,19 @@ export class CardElementPeer extends DesignerPeer {
     }
 
     updateChildren(initializeCardElement?: boolean) {
-        if (this.cardElement instanceof Adaptive.CardElementContainer) {
-            for (let i = 0; i < this.getChildCount(); i++) {
-                let existingPeer = this.getChildAt(i);
+        for (let i = 0; i < this.getChildCount(); i++) {
```

Comment:
quick question, i thought `let` is for constants and var is for variables. since `i` keeps changing its value, wouldn't `var` be more appropriate?